.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## 🎯 Changes
+
+What changes are made in this PR? Is it a feature or a bug fix?
+
+## ✅ Checklist
+
+- [ ] I have followed the steps listed in the [Contributing guide](/CONTRIBUTING.md).
+- [ ] If necessary, I have added documentation related to the changes made.
+
+## 🪪 Related Issues and Discussions
+
+Please link issues related to this pull request here.
```

Comment:
It'll likely only be one issue so we may just want to have a `closes #` at the top of the template. Like this:
```
closes #

## 🎯 Changes
...
```